### ISSUE TITLE
RC Test: Balloons And Static Electricity 1.3.0-phetiorc.1

### ISSUE THREAD

**Issue by zepumph** [phetsims/tasks#848]
As part of phetsims/phet-io#1139, we need this to help with the wrapper sonification effort with Georgia Tech.

These shas were pulled off of master because they needed to pick up many infrastructure/refactoring changes in the phet-io code.

Here is the link:
http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.3.0-phetiorc.1/wrappers/index/

@ariel-paul please assign and prioritize. It may be good to talk to either @jbphet or @emily-phet to discuss deadlines for some of the projects that this is needed for. Thanks!

**Comment by jbphet** [comment]
One of the objectives for making this release is to make it possible in the sonification wrappers to turn off sound generation for a screen when that screen is not selected or when the browser tab containing the sim is hidden.  Also, we want to be able to prevent a bunch of individual sonification sounds during reset and instead just play a single reset "earcon".  Below is a list of the issues related to the common code changes that support these features.  These should be verified as part of the testing for this release.

https://github.com/phetsims/joist/issues/421
https://github.com/phetsims/joist/issues/418
https://github.com/phetsims/phet-io-wrapper-sonification/issues/4

**Comment by EthanWJohnson** [comment]
Not seeing any issues with active or visible down here.

**Comment by phet-steele** [comment]
Testing is done, back to @zepumph. Features above were verified as (mostly) working.

**Comment by zepumph** [comment]
Closing, next rc: https://github.com/phetsims/tasks/issues/857
